crates/ink/tests/ui/pay_with_call/pass/multiple_args.rs

```diff
@@ -0,0 +1,29 @@
+#[ink::contract]
+mod contract {
+    #[ink(storage)]
+    pub struct Contract {}
+
+    impl Contract {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            Self {}
+        }
+
+        #[ink(message)]
+        pub fn message0(&self) {}
+
+        #[ink(message)]
+        pub fn message1(&self, _arg1: u8) {}
+
+        #[ink(message)]
+        pub fn message2(&self, _arg1: u8, _arg2: u8) {}
+
+        fn check_compiles(&self) {
+            ink::env::pay_with_call!(self.message0(), 0);
+            ink::env::pay_with_call!(self.message1(0), 0);
+            ink::env::pay_with_call!(self.message2(0, 0), 0);
```

Comment:
```suggestion
            ink::env::pay_with_call!(self.message2(4*64, 0), 0);
```
just to make sure it really works with expr; fun fact: it will compile regardless the value of the expr is out of bounds

Reply:
I don't get what you mean here, can you clarify?

Reply:
nevermind, just wanted to pass it a lil more interesting expr than just `0`; `4*64` evaluates to `256` which is out of `u8` bounds; still this would compile, while passing just `256` - not